decoder/microedge.go

```diff
@@ -29,20 +27,36 @@ func CheckPayloadLengthME(data string) bool {
 	return dl == 36 || dl == 32 || dl == 44
 }
 
-func DecodeME(data string, _ *LoRaDeviceDescription) (*CommonValues, interface{}) {
+func DecodeME(data string, devDesc *LoRaDeviceDescription, device *model.Device) error {
+	commonValues := &CommonValues{}
+	decodeCommonValues(commonValues, data, devDesc.Model)
+	if commonValues == nil {
+		return errors.New("invalid common values")
+	}
+
+	updateDeviceFault(commonValues.ID, commonValues.Sensor, device.UUID, commonValues.Rssi)
+
+	err := updateDevicePoint("rssi", float64(commonValues.Rssi), device)
+	if err != nil {
+		return err
+	}
+
+	err = updateDevicePoint("snr", float64(commonValues.Snr), device)
+	if err != nil {
+		return err
+	}
+
 	p := pulse(data)
+	_ = updateDevicePoint("pulse", float64(p), device)
```

Comment:
Let's move all `updateDevicePoint` at last, so it will have more better readability.